graphql/src/main/java/io/micronaut/configuration/graphql/GraphQLConfiguration.java

```diff
@@ -124,7 +124,7 @@ public static class GraphiQLConfiguration implements Toggleable {
         /**
          * The configuration name of the GraphiQL path.
          */
-        public static final String PATH = PREFIX + ".path";
+        public static final String PATH_CONFIG = PREFIX + ".path";
```

Comment:
why do we need to change this? If we don't need it, lets change it to avoid unnecessary breaking changes.